sprint-mission-2/src/main/java/com/sprint/mission/discodeit/service/jcf/JCFChannel.java

```diff
@@ -0,0 +1,86 @@
+package com.sprint.mission.discodeit.service.jcf;
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.service.ChannelService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class JCFChannel  implements ChannelService {
+
+    private final List<Channel> channelData;
+    private final JCFUser jcfUser;
+    private final JCFMessage jcfMessage;
+
+    public JCFChannel(JCFUser jcfUser,JCFMessage jcfMessage){
+        this.channelData = new ArrayList<>();
+        this.jcfUser = jcfUser;
+        this.jcfMessage = jcfMessage;
+    }
+
+    public Channel createChannel(String name, User root) {
+        Channel channel =new Channel(name, root);
+        channelData.add(channel);
+        return channel;
+    };
```

Comment:
메서드 끝 `;`는 필요없어서 제거해도 좋을 것 같아요.  
```suggestion
    }
```

Reply:
넵! 
끝에 ; 붙이면 마음이 편해서 가끔 무의식적으로 ; 붙입니다.